packages/x-charts/src/BarChart/seriesConfig/seriesProcessor.ts

```diff
@@ -35,8 +35,11 @@ const seriesProcessor: SeriesProcessor<'bar'> = (params, dataset) => {
     }
   });
 
-  const completedSeries: { [id: string]: DefaultizedProps<ChartSeries<'bar'>, 'data' | 'layout'> } =
-    {};
+  const completedSeries: {
+    [id: string]: DefaultizedProps<ChartSeries<'bar'>, 'data' | 'layout'> & {
+      stackedData: [number, number][];
+    };
+  } = {};
```

Comment:
fixing types. The `stackedData: [number, number][];` was in the `DefaultizedProps` but it's not accurate.

Since here (and in line chart) are the only placed where it's needed